sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/ChatViewModel.kt

```diff
@@ -428,6 +441,18 @@ abstract class ChatViewModel<ARGS: NavArgs>(
                         }
                     }
                 }
+            } else if (viewState is AttachmentSendViewState.PreviewGiphy) {
```

Comment:
can be folded into the previous if statement using or `||`

Reply:
There is a slight differences introduced by line 436 where we don't need to check if the file being previewed is the same one as the new one being previewed. Couldn't come up with a way to use the `||` without introducing complexity on the block of code around line 436.